newIDE/app/src/Profile/AuthenticatedUserProvider.js

```diff
@@ -97,8 +100,11 @@ export default class AuthenticatedUserProvider extends React.Component<
   _automaticallyUpdateUserProfile = true;
   _hasNotifiedUserAboutAdditionalInfo = false;
   _hasNotifiedUserAboutEmailVerification = false;
+  _cloudProjectListingPromise: Promise<
```

Comment:
should this be a boolean `_isFetchingCloudProjects` that is set to true before fetching?
I'm afraid we're adding unnecessary complexity in storing the promise

Reply:
A promise is safer there, as you can call the function `_listUserCloudProjects`  and it will always return:
- a new promise, if no call is made,
- the existing promise, if a call is in flight.